RecoHGCal/TICL/plugins/SeedingRegionByL1.cc

```diff
@@ -35,9 +36,10 @@ void ticl::SeedingRegionByL1::makeRegions(const edm::Event &ev,
   for (size_t indx = 0; indx < (*l1TrkEms).size(); indx++) {
     const auto &l1TrkEm = (*l1TrkEms)[indx];
     double offlinePt = this->tkEmOfflineEt(l1TrkEm.pt());
-    if ((offlinePt < minPt_) || (std::abs(l1TrkEm.eta()) < minAbsEta_) || (std::abs(l1TrkEm.eta()) > maxAbsEta_) ||
-        (l1TrkEm.EGRef()->hwQual() != quality_)) {
-      continue;
+    if ((offlinePt < minPt_) || (std::abs(l1TrkEm.eta()) < minAbsEta_) || (std::abs(l1TrkEm.eta()) > maxAbsEta_)) {
```

Comment:
this piece of code is a bit hard to read, I would suggest splitting conditions in dedicated booleans and then compose the final `if` accordingly (no strong push though).